pkg/cli/admin/upgrade/status/controlplane.go

```diff
@@ -41,6 +42,13 @@ type operators struct {
 	Updating int
 	// Waiting is the count of operators that have not updated its version and are with Progressing=False
 	Waiting int
+	// UpdatingClusterOperators is the collection of cluster operators that are currently being updated
+	UpdatingClusterOperators []UpdatingClusterOperator
```

Comment:
```suggestion
	Updating []UpdatingClusterOperator
```

We can replace the old `Updating` member with `len(Updating)` and then we do not need to worry about two members being in sync.

Reply:
Done.